src/models/service/transactions/mod.rs

```diff
@@ -91,8 +91,16 @@ pub struct SettingsChange {
 
 #[derive(Serialize, Debug, PartialEq)]
 #[serde(rename_all = "camelCase")]
+#[serde(tag = "type")]
 pub enum SettingsInfo {
-
+    SetFallbackHandler { handler: String },
+    AddOwnerWithThreshold { owner: String, threshold: u64 },
```

Comment:
Can we align the remove owner and add owner naming?

Reply:
I am not sure I understand what you mean by this

Reply:
One has the ending `WithThreshold` the other doesn't ... either both have it or neither

Reply:
I just matched the name according to the json. I  will take the rest off.